querybook/server/lib/metastore/base_metastore_loader.py

```diff
@@ -329,24 +329,34 @@ def _create_table_table(
             )
 
             for column in columns:
-                create_column(
+                column_id = create_column(
```

Comment:
just to double check, i thought flushing the row would generate the id?

Reply:
https://stackoverflow.com/questions/28242523/get-inserted-key-before-commit-session

Reply:
yep, flush actually works